### DISCUSSION TITLE
Make private methods of Country Service protected instead

### DISCUSSION THREAD

**Discussion by hiasl** [craftcms/commerce#2472] [Ideas]
**What are you trying to do?**
I would like to extend the Country Service to select countries by others criteria than the ones provided.
My use case is to get a list of countries which I have shipping for, which are in multiple zones.

**What's your proposed solution?**
Could you make the private methods (and variables) protected instead? _clearCaches(), _createCountryQuery + vars

**Comment by lukeholder** [comment]
https://linear.app/craftcms/issue/COM-23/review-usage-of-private-methods-convert-to-protected-where-appropriate
